lib/re_captcha/configuration.rb

```diff
@@ -33,7 +36,22 @@ def language_code(locale)
     end
 
     def valid?
-      !private_key.nil? && !public_key.nil?
+      reset_errors
+      set_errors
+      errors.empty?
+    end
```

Comment:
Well i do think it feels better:

```ruby
module ReCaptcha
  class Configuration
    include ActiveModel::Model

    API_END_POINT = 'https://www.google.com/recaptcha/'
    SKIPPED_ENVIRONMENTS = %w(test cucumber)
    LANGUAGE_TABLE = {
      'en-US' => 'en',
      'fr-FR' => 'fr',
      'es-ES' => 'es',
      'pt-PT' => 'pt-PT',
      'it-IT' => 'it',
      'en-GB' => 'en-GB',
      'de-DE' => 'de',
      'pt-BR' => 'pt-BR',
      'en-EU' => 'en-GB',
      'es-LA' => 'es-419',
      'zh-CN' => 'zh-CN',
    }

    attr_accessor :api_endpoint, :public_key,
      :private_key, :skipped_env, :language_table, :deny_on_error

    validates :public_key, presence: true
    validates :private_key, presence: true

    def initialize
      @api_endpoint   = API_END_POINT
      @skipped_env    = SKIPPED_ENVIRONMENTS
      @public_key     = ENV['RECAPTCHA_PUBLIC_KEY']
      @private_key    = ENV['RECAPTCHA_PRIVATE_KEY']
      @language_table = LANGUAGE_TABLE
      @deny_on_error  = false
    end

    def language_code(locale)
      @language_table[locale.to_s] || 'en'
    end
  end
end
```

and since it's already loaded, i see no reasons not to use it.